tests/algorithms/test_torch_export.py

```diff
@@ -123,8 +123,8 @@ def test_surgery_torchfx_eval(name, surgery_method, input):
 def test_surgery_onnx(name, surgery_method, input, tmpdir):
     """Tests onnx export and runtime"""
     pytest.importorskip("onnx")
-    import onnx
-    import onnxruntime as ort
+    import onnx  # type: ignore
```

Comment:
(same question about why to add the type ignore)

Reply:
(Same answer)